tests/net/lib/dns_sd/src/main.c

```diff
@@ -304,16 +304,30 @@ ZTEST(dns_sd, test_add_ptr_record)
 	zassert_mem_equal(actual_buf, expected_buf,
 			  MIN(actual_int, expected_int), "");
 
-	/* dns_sd_rec_is_valid failure */
-	DNS_SD_REGISTER_TCP_SERVICE(null_label,
+	/* empty instance should pass */
+	DNS_SD_REGISTER_TCP_SERVICE(null_instance,
 				NULL,
 				"_x",
 				"xx",
 				DNS_SD_EMPTY_TXT,
 				CONST_PORT);
-	zassert_equal(-EINVAL, add_ptr_record(&null_label, ttl,
+	zassert_equal(56, add_ptr_record(&null_instance, ttl,
```

Comment:
Using magic values like 56 here is not good, it is very difficult for the next developer to understand what the value means. So please adjust this a bit, for example use a separate variable that has descriptive name for this, so something like
```
int record_offset;

record_offset = add_ptr_record(&null_instance, ttl, ...);
zassert_equal(record_offset, xxx, "some explanation here...");
```
or similar, the above is just a suggestion, please adjust when/if needed.